tests/PHPStan/Rules/Arrays/data/appended-array-item.php

```diff
@@ -11,16 +11,28 @@ class Foo
 	/** @var callable[] */
 	private $callables;
 
-	public function doFoo()
+	public function accepted()
 	{
 		$this->integers[] = 4;
 		$this->integers['foo'] = 5;
-		$this->integers[] = 'foo';
```

Comment:
sorry, didn't get that fully. can you explain what you mean? because it highly annoys me that I had to do things there yeah.. I would have not touched these even, but the problem here is that invalid assignments would widen the array type and influence other errors, so I had to separate them a bit more.

Reply:
if you add 3 empty lines, line 18,19,20 (so the ones you deleted) all remaining lines in the file will not be reduced by 3

Reply:
ah simple like that, ok, good idea I guess. let me clean this up as much as I can then

Reply:
one idea, if you want to invest a few more time:

don't change the test (or only add comments after existing lines)

Reply:
yeah, it's a reasonable approach I guess. I'll revert changes, adapt the test classes to show changed behaviour and add new tests where the existing ones did not pick up issues any more as before. that should also help show the behaviour changes better 👍